corehq/apps/smsbillables/async_handlers.py

```diff
@@ -143,9 +143,15 @@ def _directed_fee(direction, backend_api_id, backend_instance_id):
 
         for backend_instance in backends:
             backend_instance = backend_instance.wrap_correctly()
+            # Skip Testing backends
             if isinstance(backend_instance, TestSMSBackend):
                 continue
 
+            # skip if backend supports selected countries only
+            supported_in_country = country_code in [int(code) for code in backend_instance.supported_countries]
```

Comment:
phone numbers are conceptually strings of digits *not* integers, so even if it worked, it'd be better to have `str(country_code) in [code for code in backend_instance.supported_countries]`, even better to have `country_code` be passed in as a string rather than an int.